tests/html.js

```diff
@@ -81,10 +81,10 @@ test('Add Errors Repeatable - Pass', t => {
 
 test('Add Required - Pass', t => {
   const param = {
-    html: '"<label for="91f79620-ba21-4a4a-a4c7-02f456129b0f">My Awesome Text Area</label><textarea id="91f79620-ba21-4a4a-a4c7-02f456129b0f" name="my-textarea--textarea" placeholder="Type something..." />test</textarea>"',
+    html: '"<label for="91f79620-ba21-4a4a-a4c7-02f456129b0f">My Awesome Text Area <mark class="mark-save">required to save</mark></label><textarea id="91f79620-ba21-4a4a-a4c7-02f456129b0f" name="my-textarea--textarea" placeholder="Type something..." />test</textarea>"',
```

Comment:
This appears to be the only test that makes sure something's been rendered correctly. I'd like to see tests for a single input, an object, and repeatables, both `save` and `publish`, as well as when `require: true` is how it was created in the content type